Documentation/project-docs/standard-platform.md

```diff
@@ -23,12 +23,14 @@ The .NET Platform Standard version represents binary portability across platform
 
 
 ## Terms
+- **.NET Platform Standard** - versioned sets of the reference assemblies that all .NET Platforms must support.
 - **PCL** - Portable Class Library
 - **Platform** - e.g. .NET Framework 4.5, Windows Phone 8.1
 - **Reference Assembly** - An assembly that contains API surface only. There is no IL in the method bodies. It is used for compilation only, and cannot be used to run. Also commonly referred to as "Contracts".
-- **Implementation Assembly** - An assembly that contains an implementation of a reference assembly. These are usually implemented in the platform itself and cannot be updated without updating the platform.
-- **.NET Platform Standard** - versioned sets of the reference assemblies.
-- **Multi-targeting** - to compile the same source code files to different target platforms, i.e. against different API sets.
+- **Implementation Assembly** - An assembly that contains an implementation of a reference assembly. These can be implemented as standalone assemblies but can sometimes be anchored by a platform and cannot be updated without updating the platform.
```

Comment:
Ah, sorry, I somehow missed it before.  I do think a bit more clarification would be helpful in that definition, something more than just a couple examples, which can be difficult to extrapolate from (and for the examples, it's not clear where a platform leaves off, e.g. is 4.5.1 a platfrom?).  But that can be done separately.